src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,24 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li className="is-active" data-cy="Tab" key={tab.id}>
+              <a
+                href="#tab-1"
+                data-cy="TabLink"
+                onClick={() => onTabSelected(tab)}
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className="block" data-cy="TabContent">
+        {activeTabId.content}
```

Comment:
The expression `activeTabId.content` assumes `activeTabId` is an object, but it should be the `id` of the active tab. You need to find the active tab object from `tabs` using `activeTabId` and then access its `content`.